src/components/Common/Header.js

```diff
@@ -53,10 +46,14 @@ export default function Header() {
             var length = res.data.data.result.length;
             var height = 0;
             setNotifications(res.data.data.result);
-            // setApprovedNotifications(res.data.data.for_me);
-            setApprovedNotifications(res?.data?.data?.result?.filter(d => d?.status == 1));
 
-            setAllRequest(res?.data?.data?.result?.filter(d => d?.status != 1));
+            setApprovedNotifications(
```

Comment:
1.Rather implementing filter implicitly in setstate , Have a habbit of storing operation in variable and than pass it to setstate.

Reply:
resolved